### DISCUSSION TITLE
Custom scheme for docker image "checkmk"

### DISCUSSION THREAD

**Discussion by seraphblade2010** [renovatebot/renovate#32001] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

CheckMK uses the following scheme for their docker-images: 2.3.0p1
Broken down into major, minor and patch:
- 2 -> Major
- 3 -> Minor
- pX -> patch

Currently when renovate-bot opens a PR for checkmk it want to "upgrade" to 2.3.0, which would be a downgrade to 2.3.0p1.
How can I configure renovate-bot to correctly display the next version?

Thanks for any help ^^

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Duplicate of https://github.com/renovatebot/renovate/discussions/31825
